packages/server/graphql/mutations/inviteToTeam.ts

```diff
@@ -63,6 +64,34 @@ export default {
       }
 
       // RESOLUTION
+      const getInviteTrustScore = async (userId: string, teamId: string) => {
+        const untrustedDomains = ['tempmail.cn']
```

Comment:
+1 put in a constant and maybe add a comment to our Slack thread so other dev understands why

Reply:
what does put in a constant mean here?

Reply:
Create a server side file like we have for `packages/client/utils/constants.ts` and add a line like:
```TypeScript
export const UNTRUSTED_DOMAINS = ['tempmail.cn']
```